code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -12,33 +12,45 @@
 
 /obj/machinery/atmospherics
 	anchored = TRUE
-	move_resist = INFINITY				//Moving a connected machine without actually doing the normal (dis)connection things will probably cause a LOT of issues.
+	move_resist = INFINITY				//Moving a connected machine without actually doing the normal (dis)connection things will probably cause a LOT of issues. (this imply moving machines with something that can push turfs like a megafauna)
```

Comment:
imply is the wrong case here, you'd want implies. On another note, what exactly are you trying to say here? This prevents megafauna from pushing pipes around?

Reply:
Yes, megafauna and other monsters can move anchored objects (like windows and walls) if they have the move var higher than the obj move_resist, this is why we have this to INFINITY. It's good to prevent cases of pushed objects (not necessarily pipes, but the bigger machines, like freezers/heaters, cryo, the HFR, etc) but it also prevents the ability to move the constructed object once placed